documentation/src/main/asciidoc/topics/proc_annotating_auto_proto_schema_builder.adoc

```diff
@@ -0,0 +1,41 @@
+[id='annotating-autoprotoschemabuilder_{context}']
+= Generating serialization context initializers
+
+After you add the ProtoStream processor and annotate your Java classes, you can add the `@AutoProtoSchemaBuilder` annotation so that {brandname} generates the Protobuf schema, accompanying marshallers, and a concrete implementation of the `SerializationContextInitializer`.
```

Comment:
"You can add the `@AutoProtoSchemaBuilder` annotation to an interface so that ..."